forelesning08/README.md

```diff
@@ -139,13 +139,13 @@ class Workout : NSObject, Coding {
 
```

Comment:
Is `NSCoding` still a thing, or should we change this to show `Codable`?

Reply:
Codable is the replacement